### PR TITLE
[UPD] Travis.yml

### PR DESCRIPTION
**Pull request by StefanRijnhart**
https://twitter.com/PedroMBaeza/status/1162448940953952256

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,38 +1,69 @@
...
   global:
-  - VERSION="10.0" TESTS="0" LINT_CHECK="0"
+  - VERSION="10.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
+  # Set this variable to some version existing as linux-generic build on
+  # https://github.com/wkhtmltopdf/wkhtmltopdf/releases
+  # if you need to install wkhtmltopdf
+  # - WKHTMLTOPDF_VERSION="0.12.5"
+  # Set the above to install a `wkhtmltopdf` version that is not the one provided
+  # by the `pov-wkhtmltopdf` repo.
+  - PHANTOMJS_VERSION="latest"
```
Comment (pedrobaeza) [inline .travis.yml]: No need for this

File: .travis.yml [hunk 2]
```diff
@@ -1,38 +1,69 @@
...
+  # The above line controls the PhantomJS version that is used for JS testing.
+  #   It is not necessary to include this value unless you are altering the default.
+  #   Use `OS` to skip the PhantomJS upgrade & use the system version instead.
+  - WEBSITE_REPO="1"
```
Comment (pedrobaeza) [inline .travis.yml]: and this

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Isn't better to leave out all the comments?

File: .travis.yml (hunk 2 above)
Comment (StefanRijnhart) [reply .travis.yml]: Removed website stuff, but I thought keeping the comments will make it easier to determine what has changed when yet another new version is committed.

Comment (pedrobaeza) [reply .travis.yml]: Well, for instructions, the template is there, and I don't see this as a diff comparison, as you usually need to fine-tune specific repos with things, and thus you might remove needed things thinking that the template has changed.

**Pull request closed without merging**